java/src/processing/mode/java/Commander.java

```diff
@@ -280,14 +281,16 @@ public Commander(String[] args) {
             JavaBuild build = new JavaBuild(sketch);
             build.build(true);
             if (build != null) {
-//              if (platformBits == 0) {
-//                platformBits = Base.getNativeBits();
-//              }
-//              if (platformBits == 0 &&
-//                Library.hasMultipleArch(platform, build.getImportedLibraries())) {
-//                complainAndQuit("This sketch can be exported for 32- or 64-bit, please specify one.", true);
-//              }
-              success = build.exportApplication(outputFolder, platform, platformBits, embedJava);
+
+                String variant;
+                if (platform == PConstants.LINUX && platformBits == 32 && Base.getNativeArch().equals("arm")) {
```

Comment:
This can be `"arm".equals(Base.getNativeArch())` so we can skip the `RuntimeException`.